druid/src/app_delegate.rs

```diff
@@ -43,6 +47,46 @@ impl<'a> DelegateCtx<'a> {
         let target = target.into().unwrap_or(Target::Global);
         self.command_queue.push_back((target, command))
     }
+
+    /// Create a new window.
+    /// `T` must represent the application state provided to [`AppLauncher::launch`].
+    ///
+    /// [`AppLauncher::launch`]: struct.AppLauncher.html#method.launch
+    pub fn new_window<T: Any>(&mut self, desc: WindowDesc<T>) {
+        if self.state_types.check_window_desc::<T>() {
+            self.submit_command(
+                Command::one_shot(commands::NEW_WINDOW, desc),
+                Target::Global,
+            );
+        } else {
+            const MSG: &str = "All application windows must represent the same type of state.";
```

Comment:
This message is confusing me.
Perhaps something like `WindowDesc<T> - T must match the application state.`